packages/frontend/store/auth.store.ts

```diff
@@ -20,37 +21,21 @@ type OnboardStatus = {
   isExploreInfoSkipped?: boolean;
 };
 
-type ConnectedState = {
-  status: 'connected';
-  address: string;
+type AuthState = {
+  status: 'initial' | 'connected' | 'disconnected';
```

Comment:
We better use an enum instead of strings. That will require refactoring in other places but as we're dealing with this now, we better push it one step further.

Reply:
Agree!